### ISSUE TITLE
Semver?

### ISSUE THREAD

**Issue by pankajpatel** [holidaypirates/ascesis-storybook#55]
Now we are adopting `semver` in `modulor-html`; shall we follow same for storybook as well?

**Comment by nogizhopaboroda** [comment]
yup, but it's already here: https://github.com/modulor-js/modulor-storybook/blob/master/package.json#L3
or you mean to use version 1.0.0 ?
